test/externalTests/trade.js

```diff
@@ -30,10 +53,13 @@ module.exports = () => (bot, done) => {
     })
   }
 
-  bot.test.setInventorySlot(36, new Item(mcData.itemsByName.emerald.id, 1, 0), (err) => {
+  bot.test.setInventorySlot(36, new Item(mcData.itemsByName.emerald.id, 64, 0), (err) => {
     assert.ifError(err)
     bot.on('entitySpawn', onEntitySpawn)
-    // The command is callibrated to be under the 100 character limit of 1.8 - 1.10:
-    bot.test.sayEverywhere(`/summon ${villagerType} 1 5 1 {Offers:{Recipes:[{buy:{id:emerald,Count:1},sell:{id:bread,Count:1}}]}}`)
-  }) */
+
+    // A command block is needed to spawn the villager due to the chat's character limit in some versions
+    bot.test.sayEverywhere(`/setblock ${commandBlockPosTxt} command_block`)
```

Comment:
Can we use `commandBlockPos.toArray().join('  ')` here? 

`commandBlockPosTxt` implies that  it's just a string. Better name would be `commandBlockPosArgs` but we're not using it any other part of code.